synapse/models/language.py

```diff
@@ -3,36 +3,43 @@
 class LangModule(s_module.CoreModule):
 
     def getModelDefs(self):
-        return (
-            ('lang', {
-
-                'types': (
-
-                    ('lang:idiom',
-                        ('str', {}),  # TODO migrate to token-searchable type
-                        {'doc': 'A subcultural idiom.'}
-                    ),
-
-                    ('lang:trans',
-                        ('str', {}),  # TODO migrate to token-searchable type
-                        {'doc': 'Raw text with a documented translation.'}
-                    ),
-
-                ),
-
-                'forms': (
-
-                    ('lang:idiom', {}, (
-                        ('url', ('inet:url', {}), {'doc': 'Authoritative URL for the idiom.'}),  # FIXME implement inet:url
-                        ('desc:en', ('str', {}), {'doc': 'English description.'}),  # TODO migrate to token-searchable type
-                    )),
-
-                    ('lang:trans', {}, (
-                        ('text:en', ('str', {}), {'doc': 'English translation.'}),  # TODO migrate to token-searchable type
-                        ('desc:en', ('str', {}), {'doc': 'English description.'}),  # TODO migrate to token-searchable type
-                    )),
+        name = 'lang'
+
+        ctors = ()
+
+        forms = (
+            ('lang:idiom', {}, (
+                ('url', ('inet:url', {}), {
+                    'doc': 'Authoritative URL for the idiom.'
+                }),
+                ('desc:en', ('str', {}), {
+                    'doc': 'English description.'
+                }),
+            )),
+
+            ('lang:trans', {}, (
+                ('text:en', ('str', {}), {
+                    'doc': 'English translation.'
+                }),
+                ('desc:en', ('str', {}), {
+                    'doc': 'English description.'
+                }),
+            )),
+        )
 
-                ),
+        types = (
+            ('lang:idiom', ('str', {}), {
+                'doc': 'A subcultural idiom.'
+            }),
 
+            ('lang:trans', ('str', {}), {
+                'doc': 'Raw text with a documented translation.'
             }),
         )
+
+        modldef = (name, {
```

Comment:
Are we changing how we're doing model defs? if so we should standardize on one way to do that through our core synapse code.

Reply:
I don't know if they all have to be 100% the same as long as they return data in the same format. It's sort of a hassle on larger models (`inet` is `return (` followed by ~1300 lines of model data)